tests/unit/Dns/Cname/Requests/UpdateRequestTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Tests\Unit\Dns\Cname\Requests;
+
+use PHPUnit\Framework\TestCase;
+use ResellerClub\Dns\Cname\Requests\UpdateRequest;
+use ResellerClub\TimeToLive;
+
+class UpdateRequestTest extends TestCase
+{
+    public function testInstantiation()
+    {
+        $ttl = new TimeToLive(TimeToLive::MINIMUM_TTL);
+        $request = new UpdateRequest(
+            'test.com',
```

Comment:
What we've been trying to do here is assigning the values to strings i.e. `$domain = 'test.com'` so it obvious to what each parameter is on the request.

Reply:
I will update this in the README.md